mirrord/agent/env/src/envs.rs

```diff
@@ -51,3 +51,10 @@ pub const POD_IPS: CheckedEnv<Vec<IpAddr>> = CheckedEnv::new("MIRRORD_AGENT_POD_
 ///
 /// Should follow `tracing` format, e.g `mirrord=trace`.
 pub const LOG_LEVEL: CheckedEnv<String> = CheckedEnv::new("RUST_LOG");
+
+/// Container id to get traffic from.
```

Comment:
Container id to "attach"/"bind"/"work with"
It's not only for traffic.

Reply:
Changed.